### ISSUE TITLE
Purchase orders when created a GRN not changing ticket status

### ISSUE THREAD

**Issue by jacobofj** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1677]
In the past when we created and finished the status of the ticket related was not changing like now. We need this work like in the past because of the way we do the orders when received.

**Comment by Repairdesk-QA** [comment]
Hello @jacobofj,
Hope you are doing well.

Can you please elaborate on your concern? Or would be great if you share the steps.
Will be happy to advise you once you share additional details.
Waiting for your response.

Thanks & Regards,
Qurat ul Ain

**Comment by jacobofj** [comment]
When we are receive the goods related to a purchase order, we create GRN and after we save it the software automatically changes the status of the ticket related to the purchase order to "In progress". We need the software dont change it

**Comment by Repairdesk-QA** [comment]
Hello @jacobofj,
Hope you are doing well.

I wanted to tell you that this is the current functionality of RepairDesk that when items in the ticket are received via GRN, the status of the ticket will be updated accordingly (to show the progress of the ticket).
We can also set the status of the ticket manually by clicking on the STATUS column in Manage Ticket or we can Create Custom Status from Store Settings-> Order Status-> Ticket Status.

I hope this will help you & if there’s anything else I can do for you, you won’t hesitate to let me know.

Thanks & Regards,
Qurat ul Ain

**Comment by jacobofj** [comment]
Yes we do it manually but is double work, and is not changing acording to the status of the ticket because we order and start repairing as soon as the parts comes and sometimes the ticket is already finished when we do the GRN, so is changing the status and is not acording to the real process in that situation.

**Comment by Uzair-RepairDesk** [comment]
@Repairdesk-QA 
Can you please check that if the ticket is completed and GRN is received later, is it still changing the ticket status?
